### PR TITLE
License

### PR DESCRIPTION
**Pull request by RoelN**
As per https://github.com/RoelN/composable-cbdt-poc/issues/2

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2019, the project authors,
+with Reserved Font Name 'Composable CBDT POC'.
```
Comment (davelab6) [inline LICENSE.txt]: No RFN

Comment (davelab6) [reply LICENSE.txt]: `Copyright (c) 2019 The Composable CBDT POC Project Authors.`

File: AUTHORS.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# This is the official list of Composable CBDT POC authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS files.
+# See the latter for an explanation.
+
+# Names should be added to this file as:
+# Name or Organization <email address>
+# The email address is not required for organizations.
+
+Google Inc.
```
Comment (davelab6) [inline AUTHORS.txt]: blank line at end of files is old school good practice

**Comment by RoelN** [comment]
@davelab6 Thanks! Addressed your feedback in https://github.com/RoelN/composable-cbdt-poc/pull/3/commits/596cff11521cbd4e73fb662c274738c38003a602

**Comment by RoelN** [comment]
@davelab6 Going ahead & merging this

**Pull request merged**